src/RIO/PrettyPrint.hs

```diff
@@ -174,3 +175,12 @@ bulletedList = mconcat . intersperse line . map (("*" <+>) . align)
 -- each.
 spacedBulletedList :: [StyleDoc] -> StyleDoc
 spacedBulletedList = mconcat . intersperse (line <> line) . map (("*" <+>) . align)
+
+-- | The 'Style' intended to be associated with a 'LogLevel'.
```

Comment:
This should have a `@since` comment.

Reply:
I have revised the pull request to add the `@since`.

Reply:
Thanks! And final request: can you include a version number bump and a changelog entry?

Reply:
I have revised the pull request to bump the version and create ChangeLog.md (I copied the format of `rio`'s).